buf.work.yaml

```diff
@@ -0,0 +1,3 @@
+version: v1
+directories:
+  - proto
```

Comment:
Do we need this bit?

Reply:
For the new version of ignite that uses sdk v0.50 it requires these to be generated when I run ignite chain serve

Reply:
Added to git ignore